### PR TITLE
Exposes the --token flag in login command help

### LINKED ISSUE #4360: oc keeps complaining about token missing without telling user how to fix it
Doing `oc login`:

```
oc login https://console.qe.openshift.com
Username: manderse@redhat.com
Login failed (401 Unauthorized)
You must obtain an API token by visiting https://api.qe.openshift.com/oauth/token/request
```

I go to the url and run `oc login --help` to get some info, it just talks about `--password`.

```
oc login https://console.qe.openshift.com --password=<secrettoken>
Username: manderse@redhat.com
Login failed (401 Unauthorized)
You must obtain an API token by visiting https://api.qe.openshift.com/oauth/token/request
```

And when I do that I get the exact same error.

it never mentions the *Actual* minimal solution:

```
oc login https://console.qe.openshift.com --token=<secrettoken>
Logged into "https://console.qe.openshift.com:443" as "manderse@redhat.com" using the token provided.

Using project "test1".

You have access to the following projects and can switch between them with 'oc project <projectname>':

  * test1 (current)
  * test2
```

Suggestions:
a) if `--password` specified use it as alias for `--token`
b) document use of `--token` in `oc login --help`
c) Add line to login that ask to go to url to say to use `oc login <hostgiventoiniitiallogin> --token=<secrettoken> `

### PR DESCRIPTION
**Pull request by fabianofranz**
Fixes #4360

### CODE REVIEW

File: pkg/cmd/cli/cli.go [hunk 1]
```diff
@@ -147,7 +147,7 @@ func NewCommandCLI(name, fullName string, in io.Reader, out, errout io.Writer) *
 	groups.Add(cmds)
 	changeSharedFlagDefaults(cmds)
 	templates.ActsAsRootCommand(cmds, groups...).
-		ExposeFlags(loginCmd, "certificate-authority", "insecure-skip-tls-verify")
+		ExposeFlags(loginCmd, "certificate-authority", "insecure-skip-tls-verify", "token")
```
Comment (liggitt) [inline pkg/cmd/cli/cli.go]: doesn't this expose it for all commands, not just login?

Comment (fabianofranz) [inline pkg/cmd/cli/cli.go]: No, notice `loginCmd` as the first argument.

Comment (liggitt) [inline pkg/cmd/cli/cli.go]: ha, blind

**Comment by liggitt** [comment]
do docs need regenerating?

**Comment by liggitt** [comment]
LGTM

**Comment by fabianofranz** [comment]
[merge]

**Pull request merged**